dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -4292,29 +4270,6 @@ Sedp::write_subscription_data_secure(
 }
 #endif
 
-DDS::ReturnCode_t
```

Comment:
We're never going to write a participant message? Don't we use these for liveliness?

Reply:
Liveliness uses a different method.  I didn't see this method being called anywhere but the CI might tell me otherwise.